shared/index.native.js

```diff
@@ -45,8 +49,8 @@ class Keybase extends Component {
 
   render () {
     return (
-      <Provider store={store}>
-        <Nav />
+      <Provider store={this.store}>
+        {dumbSheetOnly ? <DumbSheet /> : <Nav />}
```

Comment:
actually nvm, we might want to keep it since it controls the fullscreen option when dumb sheet isn't standalone